debian/syncd.init

```diff
@@ -36,6 +36,12 @@ start_mlnx()
     [ -e /dev/sxdevs/sxcdev ] || ( mkdir -p /dev/sxdevs && mknod /dev/sxdevs/sxcdev c 231 193 )
 }
 
+start_centec()
+{
+    [ -e /dev/linux_dal ] || ( insmod /usr/lib/dal.ko && mknod /dev/linux_dal c 198 0 )
```

Comment:
kernel modules are packaged into debian package and installed into base image, check here. https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/one-image.mk#L6

for the tun drivers, if you need to use it, you can insmod in the package init script.